watertap/core/zero_order_costing.py

```diff
@@ -967,6 +1110,196 @@ def cost_gac(blk):
             blk.unit_model.activated_carbon_demand[t0], "activated_carbon"
         )
 
+    def cost_hydrothermal_gasification(blk):
+        """
+        General method for costing hydrothermal gasification unit. Capital cost
+        is based on the CHG reactor and other wastewater treatment equipments.
```

Comment:
Could you expand on this comment and list out all the components that are included in the costing (since this one has a lot)?

Reply:
OK. I've added all of them.